src/main/java/com/jcraft/jsch/ChannelX11.java

```diff
@@ -45,8 +46,8 @@ class ChannelX11 extends Channel {
   static byte[] cookie = null;
   private static byte[] cookie_hex = null;
 
-  private static Hashtable<Session, byte[]> faked_cookie_pool = new Hashtable<>();
-  private static Hashtable<Session, byte[]> faked_cookie_hex_pool = new Hashtable<>();
+  private static final Map<Session, byte[]> faked_cookie_pool = new ConcurrentHashMap<>();
+  private static final Map<Session, byte[]> faked_cookie_hex_pool = new ConcurrentHashMap<>();
```

Comment:
I don't think there is any value in using a ConcurrentHashMap instead of a regular HashMap since all accesses to the faked_cookie_hex_pool are synchronized on faked_cookie_hex_pool anyway.